packages/integration/src/transform.ts

```diff
@@ -30,7 +30,7 @@ export const transformSync = ({
       configFile: false,
     });
 
-    if (!result || !result.code) {
+    if (!result || result.code === null) {
```

Comment:
Looks like it can be undefined as well. 

```suggestion
    if (!result || result.code == null) {
```